src/SequencerModule.h

```diff
@@ -22,9 +22,13 @@ class SequencerModule : public Module
     MidiSequencerPtr sequencer;
     SequencerWidget* widget = nullptr;
 
+    void setModuleId(bool fromWidget) {
+        sequencer->undo->setModuleId(this->id, fromWidget);
+    }
+
```

Comment:
new API to let the UI set the ID, in case audio is not running. This happens in 2.0 if there is not valid audio device in your patch. It's kind of inconvenient, but that's how it is.

I'm now setting the undo module ID from both the UI (in case audio isn't running) and from audio step (in case we are running headless).